core/src/main/java/org/opensearch/sql/data/model/ExprTimeValue.java

```diff
@@ -51,6 +57,21 @@ public LocalTime timeValue() {
     return time;
   }
 
+  @Override
+  public LocalDate dateValue() {
+    return LocalDate.now();
+  }
+
+  @Override
+  public LocalDateTime datetimeValue() {
+    return LocalDateTime.of(dateValue(), timeValue());
+  }
+
+  @Override
+  public Instant timestampValue() {
+    return ZonedDateTime.of(dateValue(), timeValue(), ZoneId.of("UTC")).toInstant();
+  }
+
```

Comment:
Are these in use anywhere? Please, remove if not.
`LocalDate.now();` was a blocker actually.

Reply:
Removed in 484b28c5d18cc65e8426bfcc0e72be85528e6691. Reverted changes to `DateTimeValueTest` as well.